godet.go

```diff
@@ -420,6 +427,35 @@ func (remote *RemoteDebugger) Reload() error {
 	return err
 }
 
+// CaptureScreenshot takes a screenshot, uses "png" as default format.
+func (remote *RemoteDebugger) CaptureScreenshot(format string, quality int, fromSurface bool) ([]byte, error) {
```

Comment:
I've implemented an additional function following the logic you describe, I still find it useful to get the `[]byte`, so this additional function uses the `CaptureScreenshot`, can you check?

Best!